src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PatientIdentifier.java

```diff
@@ -0,0 +1,128 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import lombok.Builder;
+import lombok.Getter;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.Split;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.NhsIdentifier;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Example PAT+RAT56:OBI+++SU:KENNEDY+FO:SARAH+TI:MISS+MI:ANGELA'
+ */
+@Builder
+public class PatientIdentifier extends Segment {
```

Comment:
is it different from PersonName Segment?

Reply:
It probably isn't - I'll try to merge them